pkg/controller/replication/replication_controller.go

```diff
@@ -222,7 +212,12 @@ func (rm *ReplicationManager) getPodControllers(pod *api.Pod) *api.ReplicationCo
 func (rm *ReplicationManager) addPod(obj interface{}) {
 	pod := obj.(*api.Pod)
 	if rc := rm.getPodControllers(pod); rc != nil {
-		rm.expectations.CreationObserved(rc)
+		rcKey, err := controller.KeyFunc(rc)
+		if err != nil {
+			glog.Errorf("Couldn't get key for replication controller %+v: %v", rc, err)
```

Comment:
I suggest using `%#v` instead of `%+v` as I'd like to have the type of rc explicitly stated in the output.

http://play.golang.org/p/xUV9dRHMxX